configs/recipes/vision/smolvlm/sft/lambda_job.yaml

```
@@ -0,0 +1,70 @@
+# Job config to full fine-tune SmolVLM Instruct, with the Oumi trainer.
+#
+# Requirements:
+#   - Set up SkyPilot GCP: https://oumi.ai/docs/en/latest/user_guides/launch/launch.html#setup
+#   - Log into WandB (`wandb login`) or disable `enable_wandb`
+#   - Log into HF: `huggingface-cli login`
+#
+# Usage:
+#   oumi launch up --config configs/recipes/vision/smolvlm/sft/gcp_job.yaml --cluster smolvlm-vision
+#
+# See Also:
+#   - Documentation: https://oumi.ai/docs/en/latest/user_guides/launch/launch.html
+#   - Config class: oumi.core.configs.JobConfig
+#   - Config source: https://github.com/oumi-ai/oumi/blob/main/src/oumi/core/configs/job_config.py
+#   - Other job configs: configs/**/*job.yaml
+
+name: smolvlm-sft-trl-train
+
+resources:
+  cloud: lambda
+  accelerators: "A100:1"
+  use_spot: false
+  # disk_size: 1000 # Disk size in GBs
+
+num_nodes: 1 # Set it to N for multi-node training.
+
+# working_dir: .
+workdir: .
```

```suggestion
working_dir: .
```

This will fix your broken CPU tests

---

just a gentle reminder: Please update this line . Otherwise, your  presubmit tests fail.

---

I deleted the file since it was for personal testing.